pkg/util/dynamichelper/discovery/cache_fallback_discovery_client_test.go

```diff
@@ -37,7 +37,7 @@ func TestVersion(t *testing.T) {
 
 	assetsVersion := strings.TrimSuffix(string(b), "\n")
 	// NOTE: This is checking for the version of the oldest supported minor stream.
```

Comment:
We should change this comment probably?
According to the comment, the version should be the oldest supported minor stream, which is 4.13.

Reply:
Thanks @bitoku , I've removed that comment as it is inaccurate the version is just a string read from a test file that has whatever people put there.

as to if we should remove the deprecated API, I'm not very sure as well, maybe a good question for TL @hlipsig @bennerv what do you think🤔